Example/SwiftUIExample/MobilePaymentsSwiftUIExample/Screens/Home/HomeView.swift

```diff
@@ -0,0 +1,220 @@
+import SquareMobilePaymentsSDK
+import SwiftUI
+
+#if canImport(MockReaderUI)
+import MockReaderUI
+#endif
+
+struct HomeView: View {
+
+    @State private var presentingPermissionsView: Bool = false
+    @State var viewModel: HomeViewModel
+    
+    private let viewHolder: MobilePaymentsSDKViewHolder = MobilePaymentsSDKViewHolder()
+    private var mobilePaymentsSDK: SDKManager { viewModel.mobilePaymentsSDK }
+    private var isIPad: Bool {
```

Comment:
Would it be possible to use UserInterfaceSizeClass instead of checking if it's iPad?

Reply:
Updated